Game/CampaignLevels.gd

```diff
@@ -0,0 +1,3 @@
+class_name CampagignLevels extends Object
```

Comment:
CampagignLevels

Reply:
I'm going to make this an issue #97 and not fix it for now